kids_books.py

```diff
@@ -0,0 +1,174 @@
+def add_book(name, author, pages):
+    book_session = connect_to()
+    book = Book(book_name=name, author=author, number_of_pages=pages, date_added=datetime.now())
+    book_session.add(book)  
+    book_session.commit()
+    print(f"Added: {book.book_name}, {book.author}, {book.number_of_pages}")
+
+
+## SUMMARY, write out the content of the library, dump file
+# vylepšit výpis půjčených knih, aby se vypsalo komu jsou půjčené
+# přidat možnost výpisu poznámek
+@kids_library.command()
+@click.option("--borrowed", default=False, is_flag=True)   # co vlastne dělá is_flag a z jakeho duvodu se tam dava
+def dump_file(borrowed):
+    book_session = connect_to()
+    qu = book_session.query(Book)   #qu (=query)
+    if borrowed:
+        qu = qu.filter(Book.lent_to != None)
+        # print(f"{Book.id} {Book.book_name} PŮJČENO uživateli {Book.lent_to}")     ## nevypíší se konkrétní hodnoty ale ty zavorky
+        #print(qu)
+        print("PŮJČENO")
+        # vypíší se knihy, které jsou půjčené, ale už se nevypisuje komu. Jak na to?
+
+    books = qu.all()
+    for book in books:
+        if book.read_yes_no:
+            print(f"{book.id}.) Název: {book.book_name}, autor: {book.author}, počet stran: {book.number_of_pages}, přidáno do knihovny: {book.date_added}, PŘEČTENO")
+        elif book.partially_read:
+            print(f"{book.id}.) Název: {book.book_name}, autor: {book.author}, počet stran: {book.number_of_pages}, přidáno do knihovny: {book.date_added}, ROZEČTENO")
+        else:
+            print(f"{book.id}.) Název: {book.book_name}, autor: {book.author}, počet stran: {book.number_of_pages}, přidáno do knihovny: {book.date_added}")
+
+
+## READ, the books that we have already read
+@kids_library.command()
+@click.option("--book_id", prompt="Insert ID of the book")
+def already_read(book_id):
+    book_session = connect_to()
+    qu = book_session.query(Book)
+    read_book = qu.filter_by(id=book_id).one()
+    read_book.read_yes_no = True
+    book_session.add(read_book)
+    book_session.commit()
+    print(f"Kniha s ID {book_id} byla přečtena")
+
+
+## PARTIALLY READ, the books where we are in the middle of the reading
+@kids_library.command()
+@click.option("--book_id", prompt="Insert ID of the book")
+def partially_read(book_id):
+    book_session = connect_to()
+    qu = book_session.query(Book)
+    read_book = qu.filter_by(id=book_id).one()
+    read_book.partially_read = True
+    book_session.add(read_book)
+    book_session.commit()
+    print(f"Kniha s ID {book_id} je rozečtená")
+
+
+## LENT TO, the book was lent to "name of person", + date of borrowing
+@kids_library.command()
+@click.option("--book_id", prompt="Enter the ID of the borrowed book")
+@click.option("--person", prompt="Enter the name of the person who borrowed the book")
+def lend(book_id, person):
+    book_session = connect_to()
+    qu = book_session.query(Book)
+    borrowed_book = qu.filter_by(id=book_id).one()
+    if borrowed_book.lent_to == None:
```

Comment:
pouziva se `is None` - jak to napovida to Sourcery v githubu, to je mimochodem dobry nastroj, ktery napovida, jak spravne napsat kod, ma plugin i primo do PyCharmu